CMakeLists.txt

```diff
@@ -39,11 +39,7 @@ EXEC_PROGRAM(
 )
 
 IF (SC_GLOBAL_PYTHON_WRAPPER)
-	IF (APPLE)
```

Comment:
Why did you remove it? Didn't it work? We had to introduce that for our OSX builds to avoid conflicts if you have a homebrew Python version installed. See Modules/FindPythonLIbsApple.cmake.